tests/transforms/test_qcut.py

```diff
@@ -0,0 +1,231 @@
+        observables contained within the graph nodes
+        """
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.432, wires=0)
+            qml.RY(0.543, wires=2)
+            qml.CNOT(wires=[0, 1])
+            qml.RZ(0.240, wires=0)
+            qml.RZ(0.133, wires=1)
+            qml.expval(obs)
+
+        g = qcut.tape_to_graph(tape)
+        nodes = list(g.nodes)
+
+        node_observables = [node for node in nodes if hasattr(node, "return_type")]
+
+        for node_obs, exp_obs in zip(node_observables, expected_obs):
+            assert node_obs.wires == exp_obs.wires
+            assert node_obs.obs.name == exp_obs.obs.name
+
+    @pytest.mark.parametrize(
+        "measurement,expected_measurement",
+        [
+            (qml.expval(qml.PauliZ(wires=[0])), "Expectation"),
+            (qml.sample(qml.PauliZ(wires=[0])), "Sample"),
+            (qml.var(qml.PauliZ(wires=[0])), "Variance"),
+            (qml.probs(wires=0), "Probability"),
+            (qml.state(), "State"),
+            (qml.density_matrix([0]), "State"),
+        ],
+    )
+    def test_measurement_conversion(self, measurement, expected_measurement):
+        """
+        Tests that measurements are correctly converted
+        """
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.432, wires=0)
+            qml.RY(0.543, wires=2)
+            qml.CNOT(wires=[0, 1])
+            qml.RZ(0.240, wires=0)
+            qml.RZ(0.133, wires=1)
+            qml.apply(measurement)
+
+        g = qcut.tape_to_graph(tape)
+        nodes = list(g.nodes)
+
+        node_observables = [node for node in nodes if hasattr(node, "return_type")]
+
+        assert node_observables[0].return_type.name == expected_measurement
+
+    @pytest.mark.parametrize(
+        "prep,expected_prep",
+        [
+            (
+                qml.BasisState(np.array([1, 1]), wires=range(2)),
+                [qml.PauliX(wires=[0]), qml.PauliX(wires=[1])],
+            ),
+            (
+                qml.QubitStateVector(np.array([1, 0, 0, 0]), wires=range(2)),
+                [qml.CNOT(wires=[0, 1]), qml.CNOT(wires=[0, 1])],
+            ),
+            # (qml.QubitDensityMatrix(np.zeros((2 ** 2, 2 ** 2)), wires=[0, 1]), "?"), #Broken
+        ],
+    )
+    def test_state_preparation(self, prep, expected_prep):
+        """
+        Tests that state preparation over mutliple wires are represented by
+        the individual operations
+        """
+
+        with qml.tape.QuantumTape() as tape:
+            qml.apply(prep)
+            qml.RX(0.432, wires=0)
+            qml.RY(0.543, wires=2)
+
+        g = qcut.tape_to_graph(tape)
+        nodes = list(g.nodes)
+
+        node_preps = nodes[:2]
+        for node_prep, expected_prep in zip(node_preps, expected_prep):
+            assert node_prep.wires == expected_prep.wires
+            assert node_prep.name == expected_prep.name
```

Comment:
Could add a test with a tape that contains two measurements. One meas could be a tensor product and the other not.

Reply:
Added thanks!